README.rst

```diff
@@ -1,148 +1,262 @@
+================================
 FIWARE Policy Manager GE - Facts
-____________________
+================================
 
 | |Build Status| |Coverage Status| |Pypi Version| |Pypi License|
 
-Description
-===========
 
-This module is part of FIWARE Policy Manager.
+This is the code repository for **FIWARE Policy Manager GE - Facts**, a server to process the incoming facts from the
+`Orion Context Broker <https://github.com/telefonicaid/fiware-orion>`__
+and publish the result into a RabbitMQ queue to be analysed by Fiware-Cloto. The facts are the result of the server
+resources consumption.
 
-Server to process the incoming facts from the `Orion Context Broker <http://catalogue.fi-ware.org/enablers/publishsubscribe-context-broker-orion-context-broker>`__.
+This project is part of FIWARE_ and `FIWARE Ops`_ suite of tools for the operation of FIWARE Lab.
```

Comment:
This project is part of FIWARE_. Check also the `FIWARE Catalogue entry for Policy Manager <http://catalogue.fiware.org/enablers/policy-manager-bosun>`__